changelogs/fragments/lowercase-response-headers.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+- client - Lowercase all header dict keys on Response initialization for better consistency across Python versions. Fix tests and table client accordingly.
```

Comment:
Can you please add a PR link at the end of the description? Thanks.

Reply:
Now it's in the description.